src/odemis/gui/main.py

```
@@ -53,12 +54,14 @@ class OdemisGUIApp(wx.App):
     """ This is Odemis' main GUI application class
     """
 
-    def __init__(self, standalone=False, file_name=None):
+    def __init__(self, standalone: bool = False, file_name: str = None, slm_alignment_mode: bool = False) -> None:
```

Ruff (0.16.3)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)